src/main/java/org/apache/commons/collections4/bloomfilter/HasherCollection.java

```diff
@@ -66,6 +66,16 @@ public HasherCollection(Hasher... hashers) {
         this(Arrays.asList(hashers));
     }
 
+    @Override
+    public boolean isEmpty() {
+        for (Hasher h : hashers) {
+            if (!h.isEmpty()) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
Bit rusty with Java & streams, but I **think** this is equivalent to:

```java
return hashers.stream().noneMatch(Hasher::isEmpty);
```

Reply:
Out of the loop, so can't comment on the addition of that method, but the rest of the code looks Ok :+1: